studio/src/aem/aem-fragments.js

```diff
@@ -179,9 +178,13 @@ class AemFragments extends LitElement {
         const fragment = await this.#aem.sites.cf.fragments.getById(
             this.searchText,
         );
-        this.#searchResult = [fragment];
+        if (!fragment || fragment.path.indexOf(getDamPath(this.path)) < 0) {
+            this.#searchResult = [];
```

Comment:
`this.#searchResult = [];` already happened in line 173, no?

Reply:
no, this is here in case some fragment is found but you are in wrong folder